doc_admin/AdminManual.adoc

```diff
@@ -1,4 +1,4 @@
-= Open Hospital 1.11.1 - Administrator’s Guide
+= Open Hospital 1.11.2 - Administrator’s Guide
```

Comment:
Shouldn't this same change be made to the top of the User's Guide?

Reply:
Yes :-)